libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClientFactory.kt

```diff
@@ -68,6 +71,57 @@ class RustMatrixClientFactory @Inject constructor(
     }
 }
 
+/**
+* Get additional user-installed certificates from the `AndroidCAStore` `Keystore`.
+*
+* The Rust HTTP client doesn't include user-installed certificates in its internal certificate
+* store. This means that whatever the user installs will be ignored.
+*
+* While most users don't need user-installed certificates some special deployments or debugging
+* setups using a proxy might want to use them.
+*
+* @return A list of byte arrays where each byte array is a single user-installed certificate
+*         in encoded form.
+*/
+fun getAdditionalCertificates(): List<ByteArray> {
```

Comment:
We should extract this method behind an interface, so we can easily have an empty implementation for release mode, or stuff like that.

Reply:
I can handle this.

Reply:
The same method should be used for release builds. While I did test this with `mitmproxy`, people/customers want to use EX with a server that is using a custom CA.